openapi.yml

```diff
@@ -58,12 +58,58 @@ paths:
           required: true
           schema:
             type: string
-        - name: limit
+        - name: content_type
+          description: Exact value found in a previous search.  Only single value is allowed like `?q=term&content_type=content_type`
```

Comment:
I would change this sentence to something a bit more clear. Not sure exactly what though. Possibly "Must use exact terminology from index, such as from a previous search result". Consider asking UX for input?

Reply:
Thanks. That seems more useful for now and I'll update to use that language. I've also opened DIP-200 to make sure we get user feedback on the docs in general.